features/support/env.rb

```diff
@@ -71,4 +71,11 @@ def get_android_unity_ndk_mapping_requests
   end
 end
 
+def get_js_source_map_requests
```

Comment:
You're following the existing pattern, but you could save quite a few lines of code in that file with a simple refactor to call down to a common function (not tested!):
```
def get_requests_with_field(name)
  Server.stored_requests.reject do |request|
    value = read_key_path(request[:body], name)
    value.nil?
  end
end
```

Reply:
Nice, this is much better!